SR2024/2024-03-12-assumption-survey.md

```diff
@@ -0,0 +1,15 @@
+---
+to: SR2022 teams, SR2023 signups, SR2024 signups
+subject: Help us understand what's involved in taking part
+---
+
+Hi all,
+
+We’re gathering some information teams, to understand the positions and opinions teams are in. If you could spare a couple of minutes to fill in this short survey, it would be much appreciated.
```

Comment:
It's the wording we used last time, I'm open to dropping it:

```suggestion
We’re gathering some information from teams, to understand the positions they are in. If you could spare a couple of minutes to fill in this short survey, it would be much appreciated.
```

Reply:
Yeah, that suggested wording reads better. I'm just not sure what "opinion" this was previously referring to.